substrate/pkg/controller/substrate/cluster/addons/tekton-dashboard.go

```diff
@@ -0,0 +1,426 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package addons
```

Comment:
Consider: https://github.com/rakyll/statik

Reply:
I am leaning towards Prateek's suggestion to not handle tekton for now since it does not have node dependency yet. but staik looks like black magic. :)